### ISSUE TITLE
not connecting to icecast2

### ISSUE THREAD

**Issue by Nurlan199206** [webcast/webcaster#7]
Hi.

i filled settings, exactly host and passwod in your demo: https://webcast.github.io/webcaster/

but i dont see mountpoint at my icecast2, it's not working.

you can help me?

**Comment by theof** [comment]
Hi !
What exactly are you using to bridge webcast to icecast ? Icecast does not speak the webcast protocol, so be sure that the connection did not fail (look it up in the dev tools - the Start Streaming button fails silently)

**Comment by Nurlan199206** [comment]
i must use liquidsoap for bridge connection?

**Comment by theof** [comment]
Indeed, you probably should use Liquidsoap. It is the only software _at my knowledge_ that supports the webcast protocol.
Be sure to configure a SSL harbor connection in LS as well, as your browser will likely block requests to unsecure endpoints from a HTTPS website ! Otherwise you could host the webcaster website on your own server, and choose not to use HTTPS. However, remember that the host/password couple will then be sent over the network in clear text

**Comment by Nurlan199206** [comment]
Please you can send me example liq file for liquidsoap, and which settings i must add in webcaster web interface?
